fhir-search/src/main/java/com/ibm/fhir/search/uri/UriBuilder.java

```diff
@@ -100,7 +100,10 @@ public String toSearchSelfUri() throws URISyntaxException {
         URI selfUri = new URI(requestUri.getScheme(), requestUri.getAuthority(), requestUri.getPath(),
                 queryString.toString(), null);
 
-        return selfUri.toString();
+        // ... originally the approach was to modify the Type.Date parameter value, however that's unreliable
+        // as a '+' is a control character which the URI code then tries to escape the escaped value.
+        String urlString = selfUri.toString();
+        return urlString.replace("+", "%2B");
```

Comment:
Originally I tried manipulating the Type.Date on 294's stream. This turns out that java.net.URI has an encoding for '%' which causes it to get encoded so you end up with `%252B` which is not ideal. There are alternatives such as switching the framework to use Guava and URLEscapers, however that is a heavy lift.  This case it converts the + to be a plus and the framework on read doesn't convert to SPACE